rotations.py

```diff
@@ -0,0 +1,415 @@
+"""Computations on the Lie group of 3D rotations."""
+
+import numpy as np
+
+GROUP_IDENTITY = np.zeros(3)
```

Comment:
why is it a zero matrix here?

Reply:
Actually this is correct! This is because the group identity for the rotations is the identity, thus the identity matrix. However, the rotation vector (also called the axis-angle vector) representing the identity matrix is the null vector.